gp/cartesian_graph.py

```diff
@@ -350,7 +362,11 @@ def forward(self, x):
 
         return locals()["_c"]
 
-    def to_sympy(self, simplify=True, parameter_names_to_values=None):
+    def to_sympy(
+        self,
+        simplify: Optional[bool] = True,
+        parameter_names_to_values: Optional[Dict[str, float]] = None,
+    ) -> List[sympy_expr.Expr]:
```

Comment:
does this return value annotation cause any problems if sympy is not available?

Reply:
see above.